mobile_app_connector/models/account_invoice.py

```diff
@@ -17,6 +17,8 @@
 class AccountInvoice(models.Model):
     _inherit = "account.invoice"
 
+    no_notifications = fields.Boolean(default=True)
```

Comment:
Please rename the field to something more understandable.
```suggestion
    avoid_mobile_donation_notification = fields.Boolean(
        default=True, help='Uncheck to send a mobile notification for donation confirmation')
```